packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1941,6 +1963,17 @@ export abstract class Chart extends Observable {
         }
     }
 
+    // The `chart.series[].tooltip.range` option is a bit different for legacy reason. This use to be
+    // global option (`chart.tooltip.range`) that could overriden the theme. But now, the tooltip range
+    // option is series-specific.
+    //
+    // To preserve backward compatiblity, the `chart.tooltip.range` theme default has been changed from
+    // 'nearest' to undefined.
+    private applySeriesTooltipDefaults(target: Series<SeriesNodeDatum, SeriesProperties<never>>) {
+        target.properties.tooltip.range ??= this.tooltip.range;
+        target.properties.tooltip.range ??= target.defaultTooltipRange;
+    }
+
```

Comment:
Note: As explained in the comment above this method, this chunk of code is responsible for compatibility. It ensure that the old (now deprecated) option `tooltip.range` continues to work.

We discussed perhaps removing the `tooltip.range` option entirely rather than deprecating it. If we decide to remove the option, then this chunk (and other parts of the PR) will become redundant and can be removed.